addons/middle-click-popup/userstyle.css

```diff
@@ -42,129 +13,96 @@ input.s3devInp {
   padding-left: 0.4em;
 }
 
-input.s3devInp:focus {
+.sa-float-bar-input:focus {
   /* Change Scratch default styles */
   box-shadow: none;
 }
 
 /* Drop down from find button */
-.s3devDDOut {
+.sa-float-bar-dropdown-out {
   display: block;
   top: -6px;
   z-index: 100;
-  width: 100%;
   max-width: 16em;
-  position: relative;
-  /*padding: 2.2em 0 0;*/
-  /*background-color: white;*/
   padding: 4px;
+  position: absolute;
+  width: 16em;
+  box-shadow: 0px 0px 8px 1px rgba(0, 0, 0, 0.3);
+  background-color: white;
+  border: none;
+  border-radius: 4px;
 }
 
 /* Drop down from find button */
-ul.s3devDD {
+.sa-float-bar-dropdown {
   display: none;
   position: relative;
   padding: 0.2em 0;
   font-size: 0.75rem;
   line-height: 1;
   overflow-y: auto;
-  min-height: 128px;
   max-height: 65vh;
```

Comment:
I think you removed the wrong style here, should keep `min-height` but remove `max-height`